app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -106,11 +106,11 @@
     %h1
       = t("admin.orders.bulk_management.loading")
 
-  %div{ :class => "sixteen columns alpha", 'ng-show' => '!RequestMonitor.loading && filteredLineItems.length == 0'}
+  %div{ class: "sixteen columns alpha", ng: {show: '!RequestMonitor.loading && filteredLineItems.length == 0', cloak: true}}
     %h1#no_results
       = t("admin.orders.bulk_management.no_results")
 
-  .margin-bottom-50{ 'ng-hide' => 'RequestMonitor.loading || filteredLineItems.length == 0' }
+  .margin-bottom-50{ ng: {hide: 'RequestMonitor.loading || filteredLineItems.length == 0', cloak: true } }
```

Comment:
I think there's a space missing between { and hide
same for show in line 109 and repeat in line 160